package.json

```diff
@@ -37,8 +37,8 @@
     "rewire": "^4.0.1"
```

Comment:
IMO:  
Since we've just added the package-lock file to this repo, we should also update the dependencies to match the latest version the ^ allows. (not talking about major bumps like nyc to v15)

Reply:
After the `cordova-common` major release, I am going to be submitting another PR to bump all dependencies to the latest anyways. Not sure it is necessary to make this type of change right now since it will be done in a few days.

Reply:
Fair enough